README.md

```diff
@@ -3,15 +3,24 @@
 </p>
 
 # Project Protocol Web
+
 Project Protocol web client. Built with Vitejs, React, and Typescript.
 
-### Getting Started
+### Getting Started - MacOS
 
 1. Clone the project
-2. From project root, run:
+2. Install and use the correct node version with:
+
+```shell
+nvm install 16.4 && nvm use 16.4
+```
+
+3. Install [pnpm](https://pnpm.io/installation#using-homebrew) via Homebrew
```

Comment:
The `pnpm` install scripts are designed to work around the node version prerequisite. Once `pnpm` is installed, it will handle node version fetching and switching based on the repository. I think this step could be condensed to just:
```suggestion
2. Install [pnpm](https://pnpm.io/installation) for your operating system. (MacOS will use either of the [POSIX Systems scripts](https://pnpm.io/installation#on-posix-systems).
```